Test/Tests/RealtimeClientPresenceTests.swift

```diff
@@ -855,6 +848,13 @@ class RealtimeClientPresenceTests: XCTestCase {
         
         mainChannel.presence.unsubscribe()
         
+        guard let transport = mainClient.internal.transport as? TestProxyTransport else {
+            fail("TestProxyTransport is not set"); return
+        }
+        
+        // Same can be achieved with sleep for more than 15 seconds for the Realtime to send LEAVE for the main client
```

Comment:
This presence message is simulating the synthesised presence `LEAVE` that Realtime would send for the `mainClient`’s original connection after not receiving a heartbeat, is that right? If so, I think it would be good to say so.

Reply:
https://github.com/ably/ably-cocoa/pull/1714/commits/a00f89bfd054b160ac5ee7a7251b36ec042838b1